app/models/search/indexer.rb

```diff
@@ -189,6 +189,8 @@ def batch
   end
 
   def index_documents
+    return if (batch = self.batch).empty?
```

Comment:
Is this to avoid calling the `batch` method twice?

Reply:
Yeah, I didn't want to generate each document twice.

Reply:
Do we ever want `batch` to re-generate its documents? Or can we do something like this in `batch`:

https://github.com/otwcode/otwarchive/blob/b5bf2e7a5cd362d1e350f2be087e8049721a2484/app/models/search/taggable_query.rb#L5

Reply:
I don't think we ever need it to regenerate the batch. I'll make the change and simplify this line.